### ISSUE TITLE
Rename `web-frame` and `web-feature` modules to `app-shell` and `app-feature`

### ISSUE THREAD

**Issue by SchnWalter** [3angleTech/webFrame#67]
### Why

Angular CLI has recently added a schematic for [app-shell](https://github.com/angular/angular-cli/pull/11590/files), this schematic can be used to generate a component/structure that is very similar to the `web-frame` terminology that we use. We should probably rename the modules, to be in line with the angular community.

### What

Change directory structure to something like:
```
src
├── app
├── app-feature
│   ├── account-feature
│   ├── profile-feature
│   └── sandbox-feature
├── app-shared
│   ├── core
│   ├── security
│   └── utils
└── app-shell
│   ├── default-shell
│   └── minimal-shell
├── assets
├── environments
└── styles
```

**Comment by SchnWalter** [comment]
Because of the way Angular CLI generates components, maybe we should add the "type of module" as prefix, so that we can use existing schematics to generate components, instead of implementing and maintaining our own:
```
app-feature-account
app-feature-profile
app-shell-default
app-shared-core?
```
What do you think?

**Comment by SchnWalter** [comment]
The same applies to pages. If you generate a page component using `ng g c`, you end up with a directory that includes the page suffix in the name, so maybe we should rename the directories, so that the structure would look like:

```
src/app-feature/app-feature-account
└── pages (page?)
    ├── account-not-activated-page
    ├── email-confirmation-page
    ├── forgot-password-page
    ├── invite-users-page
    ├── login-page
    ├── logout-page
    └── signup-page
```

NOTE: I'm using the `tree -d src/web-feature/account` command to generate the existing directory tree and I'm changing it to reflect the suggestions.

**Comment by SchnWalter** [comment]
Fixed in PR #70
